index.html

```diff
@@ -0,0 +1,144 @@
+        </div>
+        <div class="main-content">
+        <i id="hamburger" class="fa-solid fa-bars fa-2x"></i>
+        <h3 class="hello-concert">"Hello! Welcome Gen Mex"</h1>
+        <h1 class="concert-title"><b>GEN MEX CONCERT<br>AMAZON ORIGINALS<br> 2023</b></h1>
+        <p class="concert-paragraph"> A joyful event at the NeueHouse Hollywood.
+        The event brings together a new generation of mexican fresh stars who will delight 
+        the audience with some of their best hits.
+        </p>
+        <p class="date-and-time">2023.03.03(FRI) ~ 05(SUN)</p>
+        <p class="place"><b>@NeueHouse Hollywood, Columbia Square, USA</b></p>
+        </div>
+    </header>
+    <main class="main-program">
+        <h3 class="main-program-title">Main Program</h3>
+        <hr class="breakline">
+        <div class="main-container-cards">
+        <div class="main-container">
+            <div class="main-container-icon">
+                <i class="fa-solid fa-microphone fa-5x"></i>               
+            </div>
+            <h3 class="title-main-container">Opening</h3>
+            <p class="description-main-container">The youngest singers from Mexico will start this night with their awesome talent</p>
+        </div>
+        <div class="main-container">
+            <div class="main-container-icon">
+                <i class="fa-solid fa-user fa-5x"></i>            
+            </div>
+            <h3 class="title-main-container">Main Artists</h3>
+            <p class="description-main-container">Three out of our main artists for today will make us enjoy for 2 hours of their most popular songs</p>
+        </div>
+        <div class="main-container">
+            <div class="main-container-icon">
+                <i class="fa-solid fa-bacon fa-5x"></i>
+            </div>
+            <h3 class="title-main-container">Break</h3>
+            <p class="description-main-container">In this time you will enjoy with your friends while listening to diverse music, courtesy of the DJ. Let's enjoy</p>
+        </div>
+        <div class="main-container">
+            <div class="main-container-icon">
+                <i class="fa-solid fa-microphone fa-5x"></i>               
+            </div>
+            <h3 class="title-main-container">Opening</h3>
+            <p class="description-main-container">Another group of the youngest singers from Mexico will re-start this night with their awesome talent</p>
+        </div>
+        <div class="main-container">
+            <div class="main-container-icon">
+                <i class="fa-solid fa-user fa-5x"></i>            
+            </div>
+            <h3 class="title-main-container">Closing</h3>
+            <p class="description-main-container">The last main artists for today will make us enjoy for 2 hours of their most popular songs. Let's close this night!!</p>
+        </div>
+        </div>
+        <a class="see-more display-none" href="#">SEE THE WHOLE PROGRAM</a>
```

Comment:
- [ ] This link is more like a button and a link and it increases accessibility if you can allow it to refer users to the `about.html` when clicked. 
- [ ] MOreso, do you think you can use a `cursor: pointer` on this? IIt improves user experience on your page.

Reply:
Done